pkg/controller/knativeserving/minikube/minikube.go

```diff
@@ -43,3 +49,20 @@ func egress(u *unstructured.Unstructured) error {
 	}
 	return nil
 }
+
+// Check for all dependencies
+func checkDependencies(instance *servingv1alpha1.KnativeServing) error {
```

Comment:
I'm not suggesting we do it as a part of this PR, but this would be a logical place to actually install `istio-lean.yaml`. You could just create a manifestival given the upstream url and apply it. :)